gordo_components/workflow/workflow_generator/resources/argo-workflow.yml.template

```diff
@@ -631,19 +631,23 @@ spec:
         applications.gordo.equinor.com/project-version: "{{project_version}}"
     inputs:
       parameters:
+      - name: project-name
       - name: model-name
       - name: metadata
       - name: model-config
       - name: data-config
       - name: data-provider
       - name: evaluation-config
+      - name: enable-remote-logging
     container:
       image: {{ docker_registry }}/{{ docker_repository }}/gordo-model-builder:{{gordo_version}}
       env:
       - name: OUTPUT_DIR
         value: "/gordo/models/{{project_name}}/{{project_version}}/{{'{{inputs.parameters.model-name}}'}}"
       - name: MODEL_REGISTER_DIR
         value: "/gordo/models/{{project_name}}/model_register"
+      - name: PROJECT_NAME
+        value: {{ '"{{inputs.parameters.project-name}}" '}}
```

Comment:
couldn't this just be: `value: "{{  project_name }}"` directly, and not go via the input stuff?